sklearn/tests/test_pipeline.py

```diff
@@ -867,6 +867,55 @@ def test_make_pipeline():
     assert pipe.steps[2][0] == "fitparamt"
 
 
+def test_classifier_estimator_type():
```

Comment:
with https://github.com/scikit-learn/scikit-learn/pull/30122 merged, these tests should simply call `is_classifier`, ...